### PR TITLE
Activating LR and Trial Plugin (2nd draft)

### PR DESCRIPTION
**Pull request by justinchoi001**
@jrhoun, supersedes #20. Closing that PR because of rebase errors.

@jhinkey

### CODE REVIEW

**Comment by jrhoun** [comment]
@justinchoi001 - I think you are missing images from this pull. Please update when you add them.

**Comment by justinchoi001** [comment]
Added the missing images. Sorry about that.

File: docs/dxp/7.2.x/en/installation-and-upgrades/01-installing-liferay-dxp/09-trial-plugin-installation.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Trial Plugin Installation
+
+For Liferay customers who are evaluating DXP 7.2 on a trial basis, **the plugins can be accessed from within the *Apps* &rarr; *Store* (i.e., Marketplace) section of the Control Panel in your product installation**.
```
Comment (jrhoun) [inline docs/dxp/7.2.x/en/installation-and-upgrades/01-installing-liferay-dxp/09-trial-plugin-installation.md]: We say:

> the plugins can be accessed

What plugins are these? Trial Plugins? What are trial plugins? I assume that this introductory statement is missing something to the effect of saying:

> Liferay allows free 30 day trials for a Liferay DXP instance. These trials include plugins and add-ons to Liferay DXP that are typically restricted those with a DXP Subscription.

Comment (justinchoi001) [reply docs/dxp/7.2.x/en/installation-and-upgrades/01-installing-liferay-dxp/09-trial-plugin-installation.md]: According to Nicole Mak: 

> trial users should be able to download/use all EE plugins minus one that require an additional subscription
> which are Liferay Enterprise Search plugins, Liferay Mobile Device Detection, Commerce (which has a separate trial process)

**Comment by jrhoun** [comment]
Merged. Thank you.

@justinchoi001 - I did a bit of wordsmithing on the activation article and on the trial plugin article. I added additional text to the trial plugin article because out it didn't provide enough context for someone to know what it was talking about. I also added a commented out section in the "activating" article that should ideally have some links to related or relevant articles that someone might want to check out. Please review those changes. Thank you again!

**Comment by rdai10** [comment]
Hey @justinchoi001, @jrhoun ,

This pr, specifically https://github.com/liferay/liferay-learn/commit/f0ddc1588302afb1e56299c3adb96d11a89066dd will trigger a typeError when trying to build the site. I think it might have to do with the markup around the figures. Can you guys take a look at that? 

Thanks!

**Comment by Alec-Shay** [comment]
@rdai10  Sent a fix here: https://github.com/jrhoun/liferay-learn/pull/101

**Pull request closed without merging**